app/templates/buyers/award_or_cancel_brief.html

```diff
@@ -18,11 +18,24 @@
   {% endif %}
   <div class="grid-row">
     <div class="column-two-thirds">
-      <div class="single-question-page">
+      {% if already_awarded %}
+        <div class="single-question-page">
```

Comment:
Could we move this div outside of the if..else blocks to _reduce replication_ :trollface: a bit or would that break Jinja?

Reply:
Definitely. Will do that now!